persistence/src/main/scala/hmda/persistence/processing/HmdaFileValidator.scala

```diff
@@ -224,17 +208,13 @@ class HmdaFileValidator(submissionId: SubmissionId) extends HmdaPersistentActor
         replyTo ! ValidationCompletedWithErrors(submissionId)
       }
 
-    case VerifyQualityEdits(v) =>
-      persist(QualityEditsVerified(v)) { e =>
-        updateState(e)
-        sender() ! QualityEditsVerified(v)
-      }
-
-    case JustifyMacroEdit(error, j) =>
-      persist(MacroEditJustified(error, j)) { e =>
-        updateState(e)
-        sender() ! MacroEditJustified(error, j)
-      }
+    case VerifyEdits(editType, v) =>
+      if (editType == Quality || editType == Macro) {
```

Comment:
You're right that the current code will never send an edit type other than Quality or Macro. But I think safeguards like this are valuable to future-proof the code for future development. This way, the purpose of the code is clear, and it enforces the business logic, so no developers will misuse this method in the future.